src/ui-test/commandPaletteOptionsTest.ts

```diff
@@ -0,0 +1,44 @@
+import { QuickPickItem, QuickOpenBox, InputBox } from "vscode-extension-tester";
+import { getCommandPromptOptions, openCommandPrompt, typeCommandConfirm, convertArrayObjectsToText } from "./common/commonUtils";
+import { ProjectInitializer } from "./common/projectInitializerConstants";
+import { assertEqualOptions } from "./common/testUtils";
+
+
+
+const GENERAL_PROJECT_EXPECTED = [
+    'crud', 'cache', 'circuit-breaker', 'configmap', 'health-check', 'messaging', 'rest-http', 'rest-http-secured', 'istio-distributed-tracing'
+];
+
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function testCommandPaletteOffering() {
+    describe('Verify Project initializer Command palette options', () => {
+
+        let inputBox: QuickOpenBox;
+
+        it('Command palette should show proper options on the first level', async function () {
+            this.timeout(5000);
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?